.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--
+Before you report a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+**Steps to reproduce:**
+
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Output of `heapster --version`:**
+
+```
+(paste your output here)
+```
+
+**Output of `hyperkube apiserver --version=raw`:**
```

Comment:
I'm not sure if a lot of users actually using hyperkube, is it not enough information from `kubectl version`?

Reply:
sgtm. Will do.